Firmware/Marlin_main.cpp

```diff
@@ -7893,6 +7893,7 @@ SERIAL_PROTOCOLPGM("\n\n");
       - M862.3 { P"<model_name>" | Q }
       - M862.4 { P<fw_version> | Q }
       - M862.5 { P<gcode_level> | Q }
+      - M862.6 { P<active_sheet> | Q }
```

Comment:
Needs to be changed as https://reprap.org/wiki/G-code#M862.6:_Firmware_features is used by buddy firmware

Reply:
Updated to sub-code 7 instead